src/components/LoginForm/LoginForm.jsx

```diff
@@ -1,66 +1,103 @@
 import { useNavigate } from 'react-router-dom';
-import { useDispatch } from 'react-redux';
-import { string } from 'prop-types';
+import { useDispatch, useSelector } from 'react-redux';
+import { bool } from 'prop-types';
 import loginFormStyles from './LoginForm.module.scss';
 import Input from '../ui-kit/Input/Input';
 import useValidation from '../hooks/useValidation';
 import { login } from '../../services/api/login';
 import Checkbox from '../ui-kit/Checkbox/Checkbox';
+import { loginActions } from '../../services/slices/login';
 
-export default function LoginForm({ type }) {
-  const { values, handleChange } = useValidation();
+
+export default function LoginForm({ isAdmin }) {
+  const { values, handleChange, errors, validate, isValid, resetForm } = useValidation();
+  const { isLoginError, loginError } = useSelector((state) => state.loginState);
   const navigate = useNavigate();
   const dispatch = useDispatch();
 
   function onSubmit(event) {
     event.preventDefault();
-    dispatch(login(values.email, values.password)).then(() =>
-      navigate('/profile')
-    );
+    validate();
+  
+    if (isValid) {
+      dispatch(loginActions.postLoginLoading());
+      const log = login(values.email, values.password)
+      
+        log.then((response) => {
```

Comment:
```suggestion
      login(values.email, values.password)
      .then((response) => {
```
еще, log это в первую очередь как console.log. Я бы назвал loginPromise